extra/html-extra/HtmlExtension.php

```diff
@@ -110,4 +111,26 @@ public static function htmlClasses(...$args): string
 
         return implode(' ', array_unique(array_filter($classes, static function ($v) { return '' !== $v; })));
     }
+
+    /**
+     * base some base class you want to have in every matching recipes
+     * variants your recipes class
+     * compoundVariants compounds allow you to add extra class when multiple variation are matching in the same time
+     * defaultVariants allow you to add a default class when no recipe is matching
+     *
+     * @param string|list<string|null>|null $base
+     * @param array<string, array<string, string|array<string>> $variants
+     * @param array<array<string, string|array<string>>> $compoundVariants
+     * @param array<string, string> $defaultVariant
+     * @internal
+     */
+    public static function cva(array|string|null $base = null, ?array $variants = null, ?array $compoundVariants = null, ?array $defaultVariant = null): Cva
```

Comment:
We kept the nullable to avoid BC breaks in TwigComponent... but maybe can we skip them here and set default values here directly ?